osu.Game/Screens/Play/SubmittingPlayer.cs

```diff
@@ -152,6 +152,10 @@ void handleTokenFailure(Exception exception, bool displayNotification = false)
                                 Logger.Log($"Please ensure that you are using the latest version of the official game releases.\n\n{whatWillHappen}", level: LogLevel.Important);
                                 break;
 
+                            case @"invalid beatmap hash":
+                                Logger.Log($"A new version of this beatmapset is available please update. \n\n{whatWillHappen}", level: LogLevel.Important);
```

Comment:
slightly modified suggestion based on the above

```suggestion
                                Logger.Log($"This beatmap does not match the online version. Please update or redownload it.\n\n{whatWillHappen}", level: LogLevel.Important);
```

Reply:
You didn't include main purpose for an update. Should be:
"This beatmap does not match the online version. Please update or redownload it in order to submit a score.

Reply:
`{whatWillHappen}` has that covered with "Your score will not be submitted." - I don't think it's necessary to mention score submission twice here.